third_party/meerkat/Component/mmSH/android/java/src/com/samsung/android/meerkat/MeerkatServerService.java

```diff
@@ -89,7 +89,7 @@ public int onStartCommand(Intent intent, int flags, int startId) {
         sharedPreferences = applicationContext.getSharedPreferences(
                 "com.samsung.android.meerkat.CAPABILITY", Context.MODE_PRIVATE);
                 sharedPreferences.registerOnSharedPreferenceChangeListener(this);
-        setCapability(sharedPreferences.getString("capability", ""));
+        setCapability(sharedPreferences.getString("offloadjs", ""));
```

Comment:
It looks to need to combine all values of sharedPreferences?
https://developer.android.com/reference/android/content/SharedPreferences#getAll()

Reply:
It will be cleared in latest patch-set.